docker/ftest/src/test/resources/tomcat/Dockerfile

```diff
@@ -1,7 +1,6 @@
-FROM tutum/tomcat:7.0
-
+FROM tomcat:jre8
```

Comment:
I would suggest to lock it on the latest tag which we know works. Otherwise when `:jre8` tag gets updated we might run into issues of not working containers